src/styles/card.scss

```diff
@@ -0,0 +1,113 @@
+@import './variables';
+@import './stars';
+
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
margin and padding are better to reset for the list of tags you really need (* selector is not the best choice here)

And it shouldn't exist in card.scss file - move it to a separate file or at least into index.scss (same for body styles below)